content/source/docs/enterprise/saml/team-membership.html.md

```diff
@@ -6,14 +6,28 @@ sidebar_current: "docs-enterprise2-saml-team-membership"
 
 # Team Membership
 
-Terraform Enterprise can automatically add users to a team if the assertions identified in the Team Attribute Name match existing teams. This feature must be enabled and an assertion identifier must be specified in the [Team Attribute Name](./configuration.html) setting.
+Terraform Enterprise (TFE) can automatically add users to teams based on their SAML assertion, so you can manage team membership in your directory service.
 
-Users will automatically be added to teams identified in the [Team Attribute Name](./configuration.html). The team must exist first in order for the user to be added to that team. If a team is removed from the Team Attribute Name in that user's SAML assertion, that user will automatically be removed from that team.
+## Enabling and Disabling Team Membership Management
 
-## Special Case for the Owners Team
+To manage team membership via SAML, specify the name of a SAML attribute in the [Team Attribute Name](./configuration.html) setting, and ensure that the value of that atribute in the SAML assertion is a comma-separated list of team names.
 
-Unlike the other teams in an organization, the role name for the owners team must be explicitely set. Without the mapping, users will not be added as organization owners.
+Once automatic team membership is enabled, users logging in via SAML are automatically added to the teams included in their assertion, and automatically removed from any teams that _aren't_ included in their assertion. Note that this overrides any manually set team memberships; whenever the user logs in, their team membership is adjusted to match their SAML assertion.
 
-## Site admins
+Any team names that don't match existing teams are ignored; TFE will not automatically create new teams.
 
-Site admins may log in to Terraform Enterprise directly. If they are disabled on the identity provider but still enabled in Terraform Enterprise and bypass SSO, they will be able to log in.
+To disable automatic team membership, set the Team Attribute Name setting to a SAML attribute that isn't present in user assertions. If the specified attribute is absent, TFE won't automatically manage team membership on login, and you can manually add users to teams via the organization settings page.
```

Comment:
Automatic team membership management is disabled by unchecking the setting in the admin panel.